contents/newsletter/choosing-technologies.md

```diff
@@ -0,0 +1,115 @@
+  - Product engineers
+  - Founders
+  - Blog
+---
+
+From generating text to storing data to creating [ASCII drawings of cows](https://github.com/sindresorhus/cows), if you've got a problem, there is a technology out there to solve it for you.
+
+Technologies are the backbone of a product. They set the guidelines of what they can and cannot do. Choosing the right technologies makes a massive difference in whether you succeed or fail. 
+
+Unfortunately, there is no perfect way to pick a technology. Some adopt everything possible, others build anything they need themselves. Like many, we are somewhere in the middle. We aim to move fast and use technology to build something people want.
+
+Apparently, we're doing an ok job of this. Over the past year, we've shipped multiple new products like web analytics and mobile session replay on top of improving our existing products like pipelines and feature flags. Each comes with its own set of technologies. 
+
+This is what we've learned about choosing them all.
+
+## Start with a hair-on-fire problem burning problem
+
+Because of the many costs of adopting a technology, you need a good reason to do so; nice to have is not good enough. For us, this usually is one of three things:
+
+1. **Costs.** For example, our Infrastructure team projecting our AWS costs to rise well above standard benchmarks triggered us to look into solutions for our highest cost services. 
+2. **Infrastructure.** For example, reaching scaling limits for our current infrastructure. When adding "more boxes" won't fix the issue, its time to look for another solution.. 
+3. **Customers.** For example, customers demand a new feature other companies have like data exports.
+
+> **Related:** [How we decide what to build](https://newsletter.posthog.com/p/how-we-decide-what-to-build)
+
+## Find a fire-extinguishing solution to that problem
+
+The goal of technology is to solve that hair-on-fire problem. We find potential technologies that do this in one of two ways: 
+
+1. **What are other people using to solve the problem?** Technologies are by definition tools or services other people have used to solve problems. For example, part of the reason [we chose ClickHouse](/blog/how-we-turned-clickhouse-into-our-eventmansion) was that Cloudflare used it.
+
+2. **How have YOU solved this problem in the past?** There are a lot of details that are only revealed by implementing and using a technology. This means a lot of pitfalls you can avoid if you've already fallen in them once.
+
+Realistically, the evaluation process happens at the same time as figuring out the options. This is because some options can be written off immediately as they don't fit our requirements. For example, when we were looking for [scalable replacements for Postgres](/blog/how-we-turned-clickhouse-into-our-eventmansion) in our early days, we knew we needed it to be open source and self-hostable. This immediately cut down our list. 
+
+## Evaluate solutions autonomously
```

Comment:
Somewhat similar comment to earlier. This and the next section feel like a continuation of each other. Maybe it would be be better to merge them as one larger section? Either that or they just need a bit more "meat" in them to standalone.

Reply:
Added more MEAT